app/models/facet_collection.rb

```diff
@@ -20,6 +20,16 @@ def filters
     facets.select(&:filterable?)
   end
 
+  def ensure_minimum_one_open_facet(facets_to_ensure)
+    at_least_one_facet_selected = facets_to_ensure.any?(&:open_on_load?)
```

Comment:
It feels like `at_least_one_facet_selected` is acting like a method that returns a boolean.  I wonder whether a `at_least_one_facet_selected?` method would be a better choice here, rather than defining a variable?